rinad/src/ipcm/main.cc

```diff
@@ -63,6 +64,12 @@ void handler(int signum)
 		case SIGHUP:
 			rinad::IPCManager->stop();
 			break;
+		case SIGCHLD:
+			while (waitpid(WAIT_ANY, NULL, WNOHANG) > 0) {
+				LOG_DBG("Child IPC Process Daemon died, removed from process table");
+			}
+
+			break;
```

Comment:
Almost ok, but we also need to check if the exit code was -1, to print an error log.